app/models/bishop.rb

```diff
@@ -10,4 +10,12 @@ def valid_move?(new_rank, new_file)
     
     !is_move_obstructed? new_rank, new_file
   end
+
+  def image
+    if color == 'white'
+      return '&#9815;'
+    else
+      return '&#9821;'
```

Comment:
Optional: My preferred approach would probably be `color == "white" ? "&#9815;" : "&#9821;"` because it neatly reduces everything down to one line. So many options! 😄